binding/react-native/src/porcupine.tsx

```diff
@@ -80,18 +88,25 @@ class Porcupine {
    * @returns An instance of the engine.
    */
   private static async create(
+    accessKey: string,
```

Comment:
once you've changed keywords to a BuiltInKeyword type. This private create constructor can just take keyword paths, with `fromBuiltInKeywords` doing the conversion from BuiltInKeyword to string path.